site/docs/mark/area.md

```diff
@@ -54,6 +54,10 @@ Using `area` mark with one temporal or ordinal field (typically on `x`) and one
 
 <span class="vl-example" data-name="area"></span>
 
+You can set the fill color of an area to a [gradient](/docs/types.html#gradient).
```

Comment:
1) It's a bit weird that we use different data for `area` and `area_gradient` here in the same subsection (under the same `###`). 

2) area_gradient does use `overlay`, which is in the very next section.  So this example should at least come after area_overlay.

That said, I think it's probably better to just modify `area_overlay` to use gradient, all in one example so we don't add to add too many redundant examples.  (Maybe rename it to `area_gradient_overlay`)?

Reply:
I modified the docs a bit. I didn't want to replace the existing example since it's nice to show the simple `line: true` and the more sophisticated `line: {fill: 'green'}` separately.